### PR TITLE
Fix: Adding responsiveness #10146

### LINKED ISSUE #10146: [UI] Overflowing issue in Preferences page in mobile view
### Current Behavior
The dropdown selector inside the spaces is overflowing the spaces container/div on the user preferences page and has lots of space wasted.
image

### Desired Behavior
Make the spaces, extensions, and other containers/divs take the whole width of the parent container so that whitespace is utilized better.

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by kanugurajesh**
**Notes for Reviewers**

This PR fixes #10146 

- I have fixed text overflow when left bar is opened
- I have added mobile responsiveness to preference page

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [✔️] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/UserPreferences/style.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const styles = (theme) => ({
     backgroundColor: theme.palette.primary,
     color: theme.palette.secondary.text,
     '@media (max-width: 368px)': {
-      width: '180px',
+      width: '100%',
```
Comment (Yashsharma1911) [inline ui/components/UserPreferences/style.js:10]: 😬 px, thanks for fixing it, good work 👏

Comment (kanugurajesh) [reply ui/components/UserPreferences/style.js:10]: @Yashsharma1911 should i change the text size

**Comment by saurabh100ni** [comment]
Hi @kanugurajesh ,
Thanks for your PR! I encourage you to share this in the Meshery Development Meeting today at 8:30 PM IST. Share your screen, and walk us through it. It's cool to take and share feedback. Check the [Doc](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit) here to include this PR.

You're welcome at the meeting

Check to [community calendar](https://layer5.io/community/calendar) for more details of meetings at Layers.

**Comment by kanugurajesh** [comment]
@saurabh100ni I could not see the message in time so I could not attend the meeting next time  I will attend

**Comment by saurabh100ni** [comment]
Sure thing, @kanugurajesh

**Comment by leecalcote** [comment]
@kanugurajesh new screenshot with fixes included?

**Comment by kanugurajesh** [comment]
The below is for 340px screen size I have also added responsives for left nav bar.

image

image

**Comment by Yashsharma1911** [comment]
> The below is for 340px screen size I have also added responsives for left nav bar.
> 
> image
> 
> image

Thankyou however still they are too big can you reduce them down more maybe

**Comment by leecalcote** [comment]
@kanugurajesh will you be finishing this work?

**Comment by leecalcote** [comment]
@dottharun, will you assist with a final review here?

**Comment by dottharun** [comment]
@kanugurajesh Thanks for the good work.
- When testing locally below 480px, the drop-down arrow in the selector container is still too close to the textbox and sometimes overlaps.
- Please fix this issue, and there should be no more problems.

**Pull request closed without merging**